### PR TITLE
Use relative references to models in common definitions json. Remove duplicate models from each swagger collection doc

### PR DESCRIPTION
**Pull request by naveedaz**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by fearthecowboy** [comment]
The `CloningInfo` model in `CommonDefinitions` is missing the property:

``` 
  "ignoreQuotas": {
          "description": "<code>true</code> if quotas should be ignored; otherwise, <code>false</code>.",
          "type": "boolean"
        }
```

This causes a code generation change.

File: specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json [hunk 1]
```diff
@@ -15925,7 +15630,7 @@
               "type": "string",
               "readOnly": true
             },
-            "functionAppId": {
+            "function_app_id": {
```
Comment (fearthecowboy) [inline specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json]: You've changed this. This causes a code generation change.

**Review by fearthecowboy — CHANGES_REQUESTED** [review]
It looks pretty good. 

When I regenerated the code there were just the two changes, those should be fixed before proceeding.

**Comment by naveedaz** [comment]
@fearthecowboy Thanks. I have made the requested changes.

**Comment by fearthecowboy** [comment]
I've regenerated both c# and python, and it looks like this doesn't change the generated sdks, which means this is just plain cleanup, and no regeneration or changes to SDKs is required. 

(there is a cosmetic re-order in a couple python files, but nothing that changes functionality)

**Review by fearthecowboy — APPROVED** [review]

**Pull request merged**